apps/dashboard/app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -3,7 +3,6 @@ class BatchConnect::SessionContextsController < ApplicationController
   include BatchConnectConcern
   include UserSettingStore
 
-  # GET /batch_connect/<app_token>/session_contexts/new
   def new
```

Comment:
We should keep this - just to note that this method is actually tied to rendering HTML pages for that path.